src/components/tabs/js/tabsDummyWrapperDirective.js

```diff
@@ -6,38 +6,47 @@ angular
  * @private
  *
  * @param $mdUtil
+ * @param $window
  * @returns {{require: string, link: link}}
  * @constructor
- * 
+ *
  * @ngInject
  */
-function MdTabsDummyWrapper ($mdUtil) {
+function MdTabsDummyWrapper ($mdUtil, $window) {
   return {
     require: '^?mdTabs',
     link:    function link (scope, element, attr, ctrl) {
       if (!ctrl) return;
 
-      var observer = new MutationObserver(function(mutations) {
+      var observer;
+      var disconnect;
+
+      var mutationCallback = function() {
         ctrl.updatePagination();
         ctrl.updateInkBarStyles();
-      });
-
-      var config = {
-        childList: true,
-        subtree: true,
-        // Per https://bugzilla.mozilla.org/show_bug.cgi?id=1138368, browsers will not fire
-        // the childList mutation, once a <span> element's innerText changes.
-        // The characterData of the <span> element will change.
-        characterData: true
       };
 
-      observer.observe(element[0], config);
+      if('MutationObserver' in $window) {
+        var config = {
+          childList: true,
+          subtree: true,
+          // Per https://bugzilla.mozilla.org/show_bug.cgi?id=1138368, browsers will not fire
+          // the childList mutation, once a <span> element's innerText changes.
+          // The characterData of the <span> element will change.
+          characterData: true
+        };
+
+        observer = new MutationObserver(mutationCallback);
+        observer.observe(element[0], config);
+        disconnect = observer.disconnect.bind(observer);
+      } else {
+        element.on('DOMSubtreeModified', mutationCallback);
```

Comment:
@crisbeto Yes, `DOMSubtreeModified` is deprecated, but this is a fix for browsers which don't yet support `MutationObserver`. I re-implemented the functionality removed [here](https://github.com/angular/material/commit/77958a173d38aa321d9929dd8071e6ac1ef0c84c). I don't think this can be switched to an Angular watcher, as the MutationObserver watches for subtree modifications, not attributes, but debouncing sounds like a good idea. Initially the `DOMSubtreeModified` would fire up to 9x more often than `MutationObserver` in my case. Debounced it fires roughly the same amount of times as `MutationObserver` (provided MO did fire), however it fires on more occasions.